File: exam-slammr-rest-parent/exam-slammr-cloud-config-server/src/main/java/uk/co/claritysoftware/exam/slammr/cloud/config/server/ExamSlammrCloudConfigServer.java
package uk.co.claritysoftware.exam.slammr.cloud.config.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.config.server.EnableConfigServer;

/**
 * Main Spring Boot Cloud Config Server bootstrapping class
 */
@SpringBootApplication
@EnableConfigServer
public class ExamSlammrCloudConfigServer {

	public static void main(String[] args) {
		SpringApplication.run(ExamSlammrCloudConfigServer.class, args);
	}
}

File: exam-slammr-cloud-config-server/src/main/kotlin/uk/co/claritysoftware/exam/slammr/cloud/config/server/ExamSlammrCloudConfigServer.kt
package uk.co.claritysoftware.exam.slammr.cloud.config.server

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.config.server.EnableConfigServer


/**
 * Main Spring Boot Cloud Config Server bootstrapping class
 */
@SpringBootApplication
@EnableConfigServer
open class ExamSlammrCloudConfigServer

fun main(args: Array<String>) {
    SpringApplication.run(ExamSlammrCloudConfigServer::class.java, *args)
}